numojo/math/check.mojo

```diff
@@ -7,7 +7,7 @@
 
 import math
 import . _math_funcs as _mf
```

Comment:
The is a space between `.` and `_math_funcs`. Is this a correct grammar?

Reply:
fixed